lib/plugins/dev-server.js

```diff
@@ -0,0 +1,28 @@
+/*
```

Comment:
This isn't really a `dev-server` plugin... it's a "AssetOutputDisplayPlugin", which happens to be disabled when the dev-server is used.